src/backend/gporca/data/dxl/minidump/DeleteWithAfterTriggerDroppedCol.mdp

```diff
@@ -0,0 +1,336 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+create table t(i int, j int);
+insert into t values(1,2);
+alter table t drop column j;
+create function func_trigger() returns trigger as
+$$begin
+    raise notice '%', old.i;
+    return old;
+end;$$ language 'plpgsql';
+create trigger trg_t_after_delete after delete on t
+for each row execute procedure func_trigger();
+set optimizer_enable_dml_triggers to on;
+explain verbose delete from t where i = 1;
+
+delete from t where i = 1;
+
```

Comment:
What's the point of this test? There aren't any mdp tests for the triggers? If there are, it seems meaningless to write another one.

Reply:
> But there is an interesting case with after trigger on table with
dropped column. The new mdp test file was added for this case. It's suitable for
ORCA unit tests, but a real execution of a script inside gives an error. This
bug is not related to our patch; separate ticket was created.

This is exactly about mentioned test case. The given scenario was uncovered before patch.